ui/js/services/log.js

```diff
@@ -1,76 +0,0 @@
-treeherder.factory('ThLog', [
-    '$log', 'ThLogConfig',
-    function ($log, ThLogConfig) {
-        // a logger that states the object doing the logging
-
-        var ThLog = function (name) {
-            this.name = name;
-        };
-
-        /**
-         * If ``whitelist`` has values, then only show messages from those.
-         * If ``whitelist`` is empty, then skip any messages from ``blacklist`` items.
-         */
-        var whitelist = ThLogConfig.whitelist;
-        var blacklist = ThLogConfig.blacklist;
-
-        ThLog.prototype.getClassName = function () {
-            return this.name;
-        };
-
-        ThLog.prototype.debug = function () { logIt(this, $log.debug, arguments); };
-        ThLog.prototype.log = function () { logIt(this, $log.log, arguments); };
-        ThLog.prototype.warn = function () { logIt(this, $log.warn, arguments); };
-        ThLog.prototype.info = function () { logIt(this, $log.info, arguments); };
-        ThLog.prototype.error = function () { logIt(this, $log.error, arguments); };
-
-        var logIt = function (self, func, args) {
-            if ((whitelist.length && whitelist.indexOf(self.getClassName()) !== -1) ||
-                (blacklist.length && blacklist.indexOf(self.getClassName()) === -1) ||
-                (!whitelist.length && !blacklist.length)) {
-                var newArgs = Array.prototype.slice.call(args);
-                newArgs.unshift(self.getClassName());
-                func(...newArgs);
-            }
-        };
-
-        return ThLog;
-    }
-]);
-
-
-/**
- * You can use this to configure which debug lines you want to see in your
- * ``local.conf.js`` file.  You can see ONLY ``foo`` lines by adding
- * a line like:
- *
- *     ThLogConfigProvider.setWhitelist([
- *         'foo'
- *     ]);
- *
- * Note: even though this is called ThLogConfig, when you configure it, you must
- * refer to it as a ``ThLogConfigProvider`` in ``local.conf.js``.
- */
-treeherder.provider('ThLogConfig', function () {
```

Comment:
This makes the commented out example config in `sample.local.conf.js` redundant. I think we should just delete the whole local config system too (`sample.local.conf.js` and the `index.js` in the same directory as it) - what do you think?

Reply:
I totally agree.  :)